src/styles/pages/plp.scss

```diff
@@ -0,0 +1,148 @@
+@import "../vendors/include-media.scss";
+
+.plp-wrapper { margin-top: var(--space-3); }
+
+// Temporary till we have skeletons.
+.plp-temp-data-loading {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  min-height: 300px;
+}
+
+.plp-hero {
+  width: 100%;
+  margin-top: var(--space-3);
+}
+
+.plp-content-grid {
+  display: flex;
+  flex-direction: column;
+  padding-right: 0;
+  padding-left: 0;
+
+  @include media(">=notebook") {
+    display: grid;
+    grid-template-columns: repeat(12, 1fr);
+    padding-top: var(--space-4);
+    padding-right: var(--page-padding-notebook);
+    padding-left: var(--page-padding-notebook);
+    column-gap: var(--space-4);
+  }
+}
+
+.plp-filters-wrapper {
+  @include media(">=notebook") {
+    grid-row: span 2;
+    grid-column: 1 / span 3;
```

Comment:
Flip asked if we could make the filter section sticky on desktop version, so I'm adding the suggestion here:
```suggestion
    position: sticky;
    top: var(--grid-gap-0);
    grid-row: span 2;
    grid-column: 1 / span 3;
    align-self: start;
```

Reply:
[Done](https://github.com/vtex-sites/base.store/pull/240/commits/d7ff116400dbdfbbb3abce8054968f32acb80787)!
![CleanShot 2022-01-21 at 14 00 54](https://user-images.githubusercontent.com/381395/150568741-1b0c0f51-b02c-49f2-9248-839e0da69c7b.gif)